js/forum/src/components/EditUserModal.js

```diff
@@ -33,26 +34,55 @@ export default class EditUserModal extends Modal {
   }
 
   content() {
+    const topObj = this;
```

Comment:
At least for what I've tried, using this in the Button object would refer to the Button itself rather than the outer class.
(I'm not familiar with JS, maybe there are other ways that I don't know.)